### PR TITLE
ApiDoc for Cluster, Connection and Future

### PR DESCRIPTION
**Pull request by oguzdemir**
ApiDoc added to cluster.py, connection.py and future.py with Python Sphinx format.

### CODE REVIEW

File: hazelcast/cluster.py [hunk 1]
```diff
@@ -36,15 +42,34 @@ def __init__(self, config, client):
...
+        Connects to cluster.
+        """
         self._connect_to_cluster()
 
     def shutdown(self):
         pass
 
     def size(self):
+        """
+        Returns the size of the cluster.
```
Comment (asimarslan) [inline hazelcast/cluster.py]: test

File: hazelcast/connection.py [hunk 2]
```diff
@@ -226,10 +297,20 @@ def receive_message(self):
             self._builder.on_message(message)
 
     def write(self, data):
+        """
+        Writes to data to this connection when sending messages.
+
```
Comment (Serdaro) [inline hazelcast/connection.py]: Writes data to this...

File: hazelcast/future.py [hunk 3]
```diff
@@ -8,6 +8,9 @@
 
 
 class Future(object):
+    """
+    Future is used for representing a asynchronous computation' result.
+    """
```
Comment (Serdaro) [inline hazelcast/future.py]: ...an asynchronous computation result.

Comment (oguzdemir) [inline hazelcast/future.py]: Done.

**Pull request merged**